Getting-Started-with-Android-Development.md

```diff
@@ -0,0 +1,20 @@
+# Getting started with Android Development
+
+So, you want to get started with Android development? Let me take you through some basic steps you should go through.
```

Comment:
This introduction seems a bit too short. 

If you are writing this type of articles; start with what Android is, a brief history, where it stands in the market as compared to its alternatives, what the readers should already know before starting, refer to official Android documentation and community so that the reader can go ahead and research more.